src/Symfony/Component/Console/Command/Command.php

```diff
@@ -569,6 +569,8 @@ public function getSynopsis($short = false)
      * Add a command usage example.
      *
      * @param string $usage The usage, it'll be prefixed with the command name
+     *
+     * @return Command The current instance
```

Comment:
`return self` is valid as well as `$this` for phpdoc but the first one is more consistent with the future 7.1 type hinting feature.